simulation_parameters/common/difficulty_presets.json

```diff
@@ -54,6 +56,7 @@
     "SwitchSpeciesOnExtinction": false,
     "LimitGrowthRate": true,
     "FogOfWarMode": "Regular",
+    "InstantKillProtection": false,
```

Comment:
I'd say that this is kind of a core gameplay experience thing so at least for now I think it should be on for all difficulties, and players can turn it off if they decide to go for custom difficulty.